ast/builtins.go

```diff
@@ -2312,6 +2313,17 @@ var CryptoX509ParseRSAPrivateKey = &Builtin{
 	),
 }
 
+var CryptoParsePrivateKeys = &Builtin{
+	Name:        "crypto.parse_private_keys",
+	Description: "Returns a list of valid private key objects from a PEM-encoded private key",
```

Comment:
```suggestion
	Description: `Returns zero or more private keys from the given encoded string containing DER certificate data.

If the input is empty, the function will return null. The input string should be a list of one or more concatenated PEM blocks. The whole input of concatenated PEM blocks can optionally be Base64 encoded.`,
```

This is a suggestion but is based on the desc for CryptoX509ParseCertificates.